### ISSUE TITLE
Allow certain plugins to use WE without going via FAWE and ability to toggle FAWE on and off

### ISSUE THREAD

**Issue by eirikh1996** [boy0001/FastAsyncWorldedit#815]
FAWE is good to have as it makes it a lot easier to paste large schematics without risking crashing the server. However, having FAWE present is causing issues for some plugins. for example, worldedit-dependent functions in certain plugins that normally succeeds does fail if FAWE is present. Therefore, two options should be added: A way to toggle use WE without FAWE and a way to make plugins depending on WorldEdit to use WE directly, and not via FAWE.

**Comment by dordsor21** [comment]
Isn't possible with the way FAWE works, in layman's terms it "takes over" some of WE's classes, overriding them. This means WE isn't entirely loaded.
